chainnotif/bitcoindnotify/bitcoind.go

```diff
@@ -341,7 +346,8 @@ out:
 				}(msg)
 
 			case *blockEpochRegistration:
-				chainnotif.Log.Infof("New block epoch subscription")
+				chainnotif.Log.Infof(
+					"New block epoch subscription")
```

Comment:
nit: break within the logging string so it's more apparent from the formatting that it's a function call to a function that takes formatting directives (`f` suffix) that allows the different formatting?